gradoop-flink/src/main/java/org/gradoop/flink/io/impl/rdbms/tuples/Fk1Fk2Props.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.io.impl.rdbms.tuples;
+
+import org.apache.flink.api.java.tuple.Tuple3;
+import org.gradoop.common.model.impl.properties.Properties;
+
+/**
+ * Tuple for n:m relation conversion f0 : Foreign key one f1 : Foreign key two
+ * f2 : Properties of belonging table
+ */
+public class Fk1Fk2Props extends Tuple3<String, String, Properties> {
+
+  /**
+   * serial version uid
+   */
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * Foreign key one
+   */
+  private String fk1;
```

Comment:
Why add fields to tuples? Can't you use larger tuples? (You can add getters and setters there too.)